specs/certificates.table

```diff
@@ -15,7 +15,7 @@ schema([
     Column("subject_key_id", TEXT, "SKID an optionally included SHA1"),
     Column("authority_key_id", TEXT, "AKID an optionally included SHA1"),
     Column("sha1", TEXT, "SHA1 hash of the raw certificate contents"),
-    Column("path", TEXT, "Path to Keychain or PEM bundle", additional=True),
+    Column("path", TEXT, "Path to Keychain or PEM bundle", additional=True, optimized=True),
```

Comment:
I suspect there's a lot of improvement to be had in this table. But probably not trivially.